src/geonovum/style.js

```diff
@@ -12,6 +13,7 @@ function attachFixupScript(doc, version) {
     "load",
     () => {
       if (window.location.hash) {
+        // @ts-ignore
         window.location = window.location.href;
```

Comment:
I guess it would be fine if we do the following?
```suggestion
         window.location.href = window.location.href;
```

Reply:
Yeah, but then the linter complains...